README.md

```diff
@@ -32,8 +32,49 @@ If you find yourself with linting exceptions, to automatically resolve all issue
 
 ### How to run the tool
 
+First create the merge database:
+
+```
+CREATE DATABASE merge_db CHAR SET uf8;
```

Comment:
This needs to be more automated. I think we should offer two options:
1. The users place the dbs into a folder and we load them 
2. The users already have the dbs loaded and they just tell us the db names

Reply:
@psmatsinhe @edrisse

Reply:
The users can specify the db names in a settings file or as parameters

Reply:
When will option #2 happen @nthfloor ?

Reply:
imagine if the user takes this tool to their server running OpenMRS. It will already have the db loaded into MySQL - it would be strange to force the user to dump the db only for our tool to reload it again before running the transformations. 

So I think we should provide both options

Reply:
I'm aware that it will be automated. 
I thought implementing the actual merge would be on a different ticket.
I've added this instructions so we have working documentation.

Reply:
OK it makes sense. With the UI ready we can even present the existing databases so the user can select the ones he wants to work on.

Reply:
ok, I see then that is fine. We can move to a new ticket I created MEPTS-232 which will tie all this together